README.md

```diff
@@ -468,6 +482,9 @@ numArticles: org.apache.spark.rdd.RDD[Double] = MapPartitionsRDD[104] at map at
 scala> val correlation = Statistics.corr(numMentions, numArticles, "pearson")
 ```
 
+Notes: You can also query filoDB tables using Spark thrift server. Refer to [SQL/Hive Example](#sqlhive-example) for additional information regarding thrift server. 
+FiloDB logs can be viewed in corresponding spark application logs by setting appropriate setting in logback.xml. Refer to [logback configuration](http://logback.qos.ch/manual/configuration.html) and [logback appenders](http://logback.qos.ch/manual/appenders.html) for additional information regarding logging settings.
```

Comment:
Note that `logback.xml` is specific to DSE.  You might want to mention both `logback.xml` (for DSE) and `log4j.properties` (for most other Spark distros)